uber/decorators.py

```diff
@@ -110,6 +110,42 @@ def charge(self, session, payment_id, stripeToken, stripeEmail='ignored', **igno
     return charge
 
 
+def cached(func):
+    func.cached = True
+    return func
+
+
+def cached_page(func):
+    from sideboard.lib import config as sideboard_config
+    innermost = get_innermost(func)
+    func.lock = RLock()
+    @wraps(func)
+    def with_caching(*args, **kwargs):
+        if hasattr(innermost, 'cached'):
```

Comment:
So basically, we end up with a file called e.g. ``/usr/local/uber13/data/uber.site_sections.schedule.index`` which will contain the rendered page.  The filename is generated based on the module and name of the function.